src/FieldEncryptFieldSettings.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\field_encrypt\FieldEncryptFieldSettings.
+ */
+
+namespace Drupal\field_encrypt;
+
+use Drupal\encrypt\EncryptionProfileManagerInterface;
+use Drupal\field\Entity\FieldStorageConfig;
+
+/**
+ * Service class to provide extra settings to field storage.
+ */
+class FieldEncryptFieldSettings {
```

Comment:
I've reviewed this class and I honestly think we might be able to remove it. I like some of the helper functions but I think one of them will be rolled into the settings form, the others can likely be refactored...

Thoughts?